src/components/bulk-email-tool/bulk-email-form/ScheduleEmailForm.jsx

```diff
@@ -9,6 +9,7 @@ function ScheduleEmailForm(props) {
   const isMobile = useMobileResponsive();
   const { isValid, onDateTimeChange, dateTime } = props;
   const { date, time } = dateTime;
+  const descriptionDate = new Date('11/27/2023, 09:00 AM');
```

Comment:
Since we're already making the description date dynamic, maybe we can always set it in the future?

Reply:
Yes, you are right. The dynamic date will be better.
Changed to dynamic tomorrow date.